CHANGELOG.md

```diff
@@ -54,7 +65,7 @@ No (notable) API changes.
 
 ## 0.1.4 - 2020-05-23
 
-***Unreleased***
+**_Unreleased_**
```

Comment:
Ah - these changes were unintentional - I have prettier setup to run formatting on markdown files. I can revert this if you prefer.

Reply:
I think the change here was just the default settings. It just replaced italics from `*` to `_`. I'm not even sure that's something that can be configured. I didn't even notice the change till after I pushed it.

Reply:
Adding the type `markdown` to the prettier pre-commit hook config didn't change anything. If you find out how to integrate the auto-formatting of Markdown files please file a PR.